src/config/index.js

```diff
@@ -103,8 +104,8 @@ let config = {
     notSupportedBrowsers: ['ie'],
     defaultPageSize: 25,
     isMultilingual: false,
-    supportedLanguages: ['en'],
-    defaultLanguage: 'en',
+    supportedLanguages: ['de'],
+    defaultLanguage: 'de',
```

Comment:
@sneridagh Did you really mean to change the default language here?

Reply:
ups, fixing... thx!